blessed/sequences.py

```diff
@@ -40,7 +44,7 @@ def _sort_sequences(regex_seqlist):
     # types, it is feasible.
     # pylint: disable=bad-builtin
     #         Used builtin function 'filter'
-    return sorted(list(filter(None, regex_seqlist)), key=len, reverse=True)
+    return sorted(list(filter(lambda x: x is not None and x[1], regex_seqlist)), key=lambda x: len(x[1]), reverse=True)
```

Comment:
for some reason this is coming out in a different order - shouldn't they both be stable?